week1/task5/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿//5. Arrays
+//Given an integer array as an input, if the length of the array is not even write the warning message, otherwise split the array in half and add both resulting arrays together and write the result.
+
+int[] input = new[] { 1, 2, 5, 7, 2, 3, 5, 7, 9 };
+int[][] splitArray = Split(input);
+int[] result = AddArray(splitArray);
+
+int[][] Split(int[] input){ 
+
+    int len = input.Length/2;
+    int [][] result = new int[2][];
+    result[0] = new int[len];
+    result[1] = new int[len];
+
+    if(((input.Length)%2)!=0){
+        Console.WriteLine($"array lenth is not even");
+    }
```

Comment:
This check should be before you are creating the `result` and assigning the two array items. In this way you will avoid having a variable that is not used (the result var)

Reply:
I tried to do more organized the functions.
Thanks for your comments and feedback :-)

Reply:
I tried to do more organized the functions.
Thanks for your comments and feedback 👍